Source/Parser/wp_parser.lex.cpp

```diff
@@ -521,13 +521,16 @@ int yy_flex_debug = 0;
 #define YY_MORE_ADJ 0
 #define YY_RESTORE_YY_MORE_OFFSET
 char *yytext;
+#line 1 "wp_parser.l"
+#line 3 "wp_parser.l"
```

Comment:
we need to remove these lines again because they cause some problems with nvcc.
will do a follow-up PR :)